pkg/booleanpolicy/default_policies_test.go

```diff
@@ -1868,9 +1868,9 @@ func policyWithSingleKeyValue(fieldName, value string, negate bool) *storage.Pol
 }
 
 func policyWithSingleFieldAndValues(fieldName string, values []string, negate bool, op storage.BooleanOperator) *storage.Policy {
-	return policyWithGroups(storage.EventSource_NOT_APPLICABLE, &storage.PolicyGroup{FieldName: fieldName, Values: sliceutils.Map(values, func(val *string) *storage.PolicyValue {
-		return &storage.PolicyValue{Value: *val}
-	}).([]*storage.PolicyValue), Negate: negate, BooleanOperator: op})
+	return policyWithGroups(storage.EventSource_NOT_APPLICABLE, &storage.PolicyGroup{FieldName: fieldName, Values: sliceutils.Map(values, func(val string) *storage.PolicyValue {
+		return &storage.PolicyValue{Value: val}
```

Comment:
It's because one has to decide on pointer or value when using generics, and pointer is more natural. If the need arises to avoid copying due to performance concerns, we can introduce a separate `FindMatchingPtr` version. However, none of the occurrences justify this: in this case, that's because `string` copies in golang are cheap (due to the immutability of strings) and also because this is test code.